src/application/worker/store/modules/modules.js

```diff
@@ -325,6 +325,11 @@ const actions = {
     { state, commit },
     { moduleId, prop, data, group, groupName, writeToSwap }
   ) {
+    if (state.active[moduleId] === undefined) {
```

Comment:
I'd probably use `if (!state.active[moduleId]) {` here as that's more consistent with the rest of the codebase - most of the time we don't really care if it's actually undefined, we just need to know something is there.

Reply:
Fixed!

Reply:
This was also done!